distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc

```diff
@@ -1,61 +1,71 @@
 :_mod-docs-content-type: ASSEMBLY
 include::_attributes/common-attributes.adoc[]
-[id="distributed-tracing-rn-3-0"]
-= Release notes for {DTProductName} 3.0
-:context: distributed-tracing-rn-3-0
+[id="distributed-tracing-rn-3-1"]
+= Release notes for {DTProductName} 3.1
+:context: distributed-tracing-rn-3-1
 
 toc::[]
 
 include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
 
-[id="component-versions_distributed-tracing-rn-3-0"]
-== Component versions in the {DTProductName} 3.0
+[id="distributed-tracing-rn_3-1_component-versions"]
+== Component versions in the {DTProductName} 3.1
 
 [options="header"]
 |===
 |Operator |Component |Version
+
 |{JaegerName}
 |Jaeger
-|1.51.0
+|1.53.0
 
-|xref:../../otel/otel-release-notes.adoc[{OTELName}]
+//for each new release, update the release number in the xref path on the following line
+|xref:../../otel/otel_rn/otel-rn-3.1.adoc[{OTELName}]
 |OpenTelemetry
-|0.89.0
+|0.93.0
 
 |{TempoName}
 |Tempo
-|2.3.0
+|2.3.1
+
 |===
 
 // Jaeger section
-[id="jaeger-release-notes_distributed-tracing-rn-3-0"]
+[id="distributed-tracing-rn_3-1_jaeger-release-notes"]
```

Comment:
Can we movo tempo RN to the top of the page?

Reply:
Great idea! Sure, let me swap them!